tools/images/ubuntu/Dockerfile

```diff
@@ -11,12 +11,12 @@ RUN truncate -s0 /tmp/preseed.cfg; \
     echo "tzdata tzdata/Zones/Europe select London" >> /tmp/preseed.cfg; \
     apt-get update && \
     apt-get install -y python3 python3-dev python3-pip time \
-    curl unzip git locales pkg-config zlib1g-dev psmisc awscli \
```

Comment:
awscli doesn't appear to be available in noble - it's probably a snap or smthn. I don't think we need it any more anyway